app/controllers/staff/teammates_controller.rb

```diff
@@ -62,4 +62,11 @@ def group_count(group)
     team_counts
   end
 
+  def require_contact_email
+    if @event.contact_email.empty?
```

Comment:
Curious about the use of @event when you've got current_event referenced in the rest of the controller.